repos/system_upgrade/el7toel8/actors/biosdevname/tests/test_biosdevname.py

```diff
@@ -0,0 +1,128 @@
+from mock import mock_open, patch
+import pytest
+import pyudev
+
+from leapp.exceptions import StopActorExecutionError
+from leapp.libraries.actor import library
+from leapp.libraries.stdlib import api
+from leapp.models import Interface, PCIAddress
+
+
+class logger_mocked(object):
```

Comment:
```suggestion
class LoggerMocked(object):
```

I noticed that we are using this class in multiple tests now, so we should probably move it to library but that's a task for different PR.